hibernate-core/src/main/java/org/hibernate/internal/AbstractSharedSessionContract.java

```diff
@@ -730,6 +739,83 @@ public QueryImplementor createQuery(String queryString) {
 		}
 	}
 
+	private CriteriaCompiler criteriaCompiler;
```

Comment:
I'm not a big fan of adding a field in the middle of the code.

Reply:
I just moved this code from another class, so I thought I'd keep the same style. I can move the field to the top if you want?

Reply:
Done